src/clincoded/static/components/variant_central/interpretation/segregation.js

```diff
@@ -63,8 +233,210 @@ var CurationInterpretationSegregation = module.exports.CurationInterpretationSeg
         );
     },
 
+    renderMasterTable() {
+        return <MasterEvidenceTable
+                    evidence_arr = {this.getAllInterpretations()}
+                    affiliation = {this.props.affiliation}
+                    session = {this.props.session}
+                    viewOnly = {this.state.data && !this.state.interpretation}
+                    deleteEvidenceFunc = {this.deleteEvidenceFunc}
+                    evidenceCollectionDone = {this.evidenceCollectionDone}
+                    canCurrUserModifyEvidence={this.canCurrUserModifyEvidence}
+                >
+                </MasterEvidenceTable>
+    },
+
+    getAllInterpretations() {
+        let relevantEvidenceListRaw = [];
+        if (this.props.data && this.props.data.associatedInterpretations) {
+            this.props.data.associatedInterpretations.map(interpretation => {
+                if (interpretation.extra_evidence_list) {
+                    interpretation.extra_evidence_list.forEach(extra_evidence => {
+                        // temporary codes
+                        if (extra_evidence.category === 'case-segregation') {
+                            relevantEvidenceListRaw.push(extra_evidence);
+                        }
+                    });
+                }
+            });
+        }
+        let relevantEvidenceList = _(relevantEvidenceListRaw).sortBy(evidence => {
+            return evidence.date_created;
+        }).reverse();
+        return relevantEvidenceList;
+    },
+
     render() {
+        let that = this;
+        function getVariantUUID() {
```

Comment:
If you make this an arrow function, you don't need to use `that`
```suggestion
        const getVariantUUID = () => {
        ...
        };
```